src/plugins/media/nodeviews/mediaNodeView/media.tsx

```diff
@@ -18,12 +18,17 @@ import { Node as PMNode } from 'prosemirror-model'
 import { CellSelection } from '@atlaskit/editor-tables/cell-selection'
 import { EditorView } from 'prosemirror-view'
 import React, { Component } from 'react'
-import { ProsemirrorGetPosHandler, ReactNodeProps } from '../../../../nodeviews'
-import { setNodeSelection, setTextSelection } from '../../../../utils'
+import {
+  ProsemirrorGetPosHandler,
+  ReactNodeProps
+} from '@atlaskit/editor-core/nodeviews'
+import { setNodeSelection, setTextSelection } from '@atlaskit/editor-core/utils'
 import { stateKey as mediaStateKey } from '../../pm-plugins/plugin-key'
 import { MediaPluginState } from '../../pm-plugins/types'
 import { MediaCardWrapper } from '../styles'
 import { MediaOptions } from '../../types'
+import CozyClient from 'cozy-client'
+import ServiceClient from 'lib/collab/stack-client'
```

Comment:
Since this is a “fork” can you write: 

/* override by cozy */ /*end override*/

Reply:
Fixed (removed)